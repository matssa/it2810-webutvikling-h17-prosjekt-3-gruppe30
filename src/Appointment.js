import React from "react";
import DatePickerS from "./Datepicker.js";
import TimePickerS from "./Datepicker.js";
import InputTodo from "./InputTodo.js";
import FormAppointment from "./FormAppointment.js";

const Appointment = () => (
  <div>
    <h1>Appointment page!</h1>
    <FormAppointment />
  </div>
)

export default Appointment;
