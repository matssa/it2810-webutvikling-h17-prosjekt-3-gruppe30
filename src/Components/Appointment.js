import React from "react";
import FormAppointment from "./FormAppointment.js";

const Appointment = () => (
  <div>
    <h1>Appointment page!</h1>
    <FormAppointment />
  </div>
)

export default Appointment;
