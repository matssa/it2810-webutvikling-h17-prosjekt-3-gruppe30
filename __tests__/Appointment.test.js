import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import Appointment from '../src/Components/Appointment.js';

describe('<Appointment/>, functions', () => {
	it('should match to previous snapshots', () => {
		const component = shallow(<Appointment />);
		const tree = toJson(component);
		expect(tree).toMatchSnapshot();
	});
});
