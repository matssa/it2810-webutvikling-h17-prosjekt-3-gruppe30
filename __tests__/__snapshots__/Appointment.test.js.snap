// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Appointment/>, functions should match to previous snapshots 1`] = `
<div>
  <h1>
    Appointment page!
  </h1>
  <FormAppointment />
</div>
`;
